statsmodels/stats/power.py

```diff
@@ -209,6 +209,8 @@ def solve_power(self, **kwds):
         #TODO: maybe use explicit kwds,
         #    nicer but requires inspect? and not generic across tests
         #    I'm duplicating this in the subclass to get informative docstring
+        kwds['power'] = power
+        kwds['alpha'] = alpha
```

Comment:
just spot checking.

There is a todo for thinking about how to do this and we are not "fixing" or changing this as pep-8/flake cleanup.

Reply:
this is the only change besides reformatting in code that is in "public" use

Reply:
> There is a todo for thinking about how to do this 

That is effectively meaningless.  It's been sitting there for 6 years.

> and we are not "fixing" or changing this as pep-8/flake cleanup.

This code is bad and it should feel bad.  It only doesn't raise because of a non-obvious quirk of how `**kwargs` is treated in the namespace.  This doesn't change the behavior of the function, just makes it clearer for readers.

----
Please take a look at these, which do not have solutions that are obvious to me:

```
./examples/python/ols.py:197:5: F821 undefined name 'norm_x'
./examples/python/ols.py:198:19: F821 undefined name 'norm_x'
./examples/python/ols.py:198:28: F821 undefined name 'norm_x'
./statsmodels/distributions/empirical_distribution.py:143:63: F821 undefined name 'ival'
./statsmodels/miscmodels/count.py:127:30: F821 undefined name 'result'
./statsmodels/miscmodels/count.py:130:38: F821 undefined name 'params'
./statsmodels/sandbox/infotheo.py:355:12: F821 undefined name 'condent'
./statsmodels/sandbox/infotheo.py:355:50: F821 undefined name 'condent'
./statsmodels/sandbox/panel/correlation_structures.py:133:20: F821 undefined name 'self'
./statsmodels/sandbox/panel/correlation_structures.py:134:9: F821 undefined name '_x'
./statsmodels/sandbox/panel/correlation_structures.py:134:22: F821 undefined name '_x'
./statsmodels/sandbox/panel/correlation_structures.py:136:12: F821 undefined name '_x'
./statsmodels/sandbox/panel/correlation_structures.py:136:15: F821 undefined name 'self'
./statsmodels/sandbox/panel/correlation_structures.py:165:18: F821 undefined name 'r'
./statsmodels/sandbox/panel/correlation_structures.py:167:30: F821 undefined name 'r'
./statsmodels/sandbox/panel/correlation_structures.py:168:15: F821 undefined name 'r'
./statsmodels/sandbox/panel/correlation_structures.py:168:23: F821 undefined name 'r'
./statsmodels/sandbox/panel/correlation_structures.py:205:16: F821 undefined name 'cov2corr'
./statsmodels/sandbox/panel/correlation_structures.py:205:25: F821 undefined name 'corr'
./statsmodels/sandbox/regression/runmnl.py:157:23: F821 undefined name 'logsumexpxp'
./statsmodels/sandbox/regression/runmnl.py:164:18: F821 undefined name 'branches'
./statsmodels/sandbox/regression/runmnl.py:165:25: F821 undefined name 'loglike_leafbranch'
./statsmodels/sandbox/regression/runmnl.py:172:30: F821 undefined name 'sumexpxb'
./statsmodels/stats/regularized_covariance.py:35:23: F821 undefined name 'ind'
./statsmodels/stats/regularized_covariance.py:79:23: F821 undefined name 'ind'
./statsmodels/tsa/kalmanf/kalmanfilter.py:335:20: F821 undefined name '_univariatefilter'
./tools/update_web.py:309:34: F821 undefined name 'new_branch_dir'
```

cc: @TomAugspurger @jseabold @ChadFulton @cclaus @bashtage